test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/CS/Source/CodeBlockWithTextElement.cshtml

```diff
@@ -1,4 +1,4 @@
 ﻿@{
-    var a = 1; <text>@a</text>
```

Comment:
Hmm these tests are all super odd. Since you're touching it can you modify these text tags to have more than just C# expressions? Purpose of them is to output literal markup but we don't actually verify that. Doing this will also make it easier to visualize your changes